bindings/setup.py

```diff
@@ -20,6 +22,9 @@
     sources=fw_sources + ["bin/cffirmware_wrap.c"],
     extra_compile_args=[
         "-O3",
+        # The following flags are also used for compiling the actual firmware
+        "-fno-strict-aliasing",
+        "-Wno-address-of-packed-member",
```

Comment:
I am not sure how all of this newfangled things you kids use work. But. Should we use the same strategy as as in our Makefile here?

In that case we might also need to add:

```
CFLAGS += -Wno-array-bounds
CFLAGS += -Wno-stringop-overread
CFLAGS += -Wno-stringop-overflow
```

Or do we cross that bridge when needed?

Reply:
My design goal was to use as few extra settings as possible and at most those that we use for the regular compilation (i.e., the Python should use a subset of the flags of the real firmware). Those are the two that I needed so far, in order to avoid compiler warnings. I'd suggest that we add the other three flags only when needed.